CMakeLists.txt

```diff
@@ -40,17 +58,40 @@ set(NOVELRT_VULKAN_VERSION "1.3.211" CACHE STRING "Vulkan version")
 set(NOVELRT_VULKAN_MEMORY_ALLOCATOR_VERSION "2.3.0" CACHE STRING "VulkanMemoryAllocator version")
 set(NOVELRT_VULKAN_SDK_PATH "" CACHE STRING "Directory for NovelRT to find the Vulkan SDK (mainly for macOS support)")
 
-find_package(Doxygen ${NOVELRT_DOXYGEN_VERSION}
-  COMPONENTS dot
-)
+if(NOVELRT_PIE)
+  set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+endif()
 
-if(NOT DOXYGEN_FOUND)
-  message("Doxygen ${NOVELRT_DOXYGEN_VERSION} was not found, building documentation is disabled.")
-  set(NOVELRT_BUILD_DOCUMENTATION OFF)
+# Remove unused symbols in non-debug mode.
+# This is useful even with LTO (-84 KiB with MinSizeRel).
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+  # For some reason, adding to CMAKE_CXX_FLAGS results in a slightly smaller
+  # binary than using `add_compile/link_options`
```

Comment:
I believe CMake actually makes an `install/strip` target if you have install targets in your build script, which should automatically do this: https://cmake.org/pipermail/cmake/2013-March/054021.html